conf.d/halostatue_fish_docker.fish

```diff
@@ -8,7 +8,7 @@ function _halostatue_fish_docker_print_containers --description 'Print a list of
         case running
             set filter --filter status=running
         case stopped
-            set filter --filter status=exited --filter status=stopped
+            set filter --filter status=exited
```

Comment:
Is this correct? The original code that I forked this from had both `status=exited` and `status=stopped`.

Reply:
`stopped` does not appear to be a valid status (see https://docs.docker.com/engine/reference/commandline/ps/#filtering) and generated an error for me.